src/styles/base/_reset.scss

```diff
@@ -0,0 +1,21 @@
+html,
+body,
+div,
+h1,
+h2,
+h3,
+p,
+span,
+ul,
+li,
+button {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  font-family: Roboto, system-ui, sans-serif;
```

Comment:
Consider using a variable for the font-family to maintain consistency and make it easier to update in the future. You already have a $fontFamily variable defined.